stackslib/src/net/inv/nakamoto.rs

```diff
@@ -113,6 +113,8 @@ pub struct InvGenerator {
     tip_ancestor_search_depth: u64,
     /// count cache misses for `processed_tenures`
     cache_misses: u128,
+    /// Disable caching (test only)
+    no_cache: bool,
```

Comment:
For safety, should we put this behind a `#[cfg(test)]` flag?

Reply:
Sure